docs/Configure.md

```diff
@@ -0,0 +1,176 @@
+# Configuring Eclair
+
+Eclair reads its configuration file, and writes its logs, to `~/.eclair` by default.
+You can change this behavior with the `eclair.datadir` parameter:
+
+```sh
+eclair-node.sh -Declair.datadir="/path/to/custom/eclair/data/folder"
+```
+
+## Change your node's configuration
+
+The first step is to **actually create the configuration file**.
+Go to `eclair.datadir` and create a file named `eclair.conf`.
+The dncoding should be UTF-8.
```

Comment:
```suggestion
The encoding should be UTF-8.
```

Reply:
I clearly couldn't decide between encoding and decoding.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1575/commits/463f39a919ef93f525095441f6449ba1090612a7